source/sass/patterns/molecules/figure.scss

```diff
@@ -0,0 +1,32 @@
+@import "../../variables/breakpoint";
+@import "../../mixins/spacing";
+@import "../../mixins/typography";
+
+.figure {
+  @include block-spacing($end: $baselinegrid-space-medium);
+}
+
+.figure__image {
+  @include block-spacing($start: $baselinegrid-space-extra_small, $end: $baselinegrid-space-extra_small);
+  @include margin(12px, inline);
+  border: 1px solid $color-text-dividers;
+  display: block;
+
+}
+
+.figure__heading {
+  @include caption-typography();
```

Comment:
Need `@include block-spacing($end: 0)` too.

Reply:
Wondering if should use

```scss
@include inherit-all($ensure: font-family font-size line-height) {
    font-weight: 600;
}
```

rather than `caption-typography`?

Reply:
Would rather keep a named typographical mixin to match the overall approach.